source/arriving/welcome.rst

```diff
@@ -5,86 +5,35 @@ Onboarding new students
 
 Important rooms and places
 """"""""""""""""""""""""""
-Chantal Proulx, Marie-Pier Côté and generally the `Faculty’s offices <https://www.usherbrooke.ca/informatique/nous-joindre/personnel>`_ are important and you should know where they are. Ask someone in the lab or Maxime to guide you.
+The Sherbrooke Connectivity Imaging Laboratory is located in the Faculty of Sciences of the University of Sherbrooke (room number **D6-0051**). Other important rooms include the Faculty’s offices:
```

Comment:
Maybe add François and Maxime rooms?

Reply:
According to the UdeS page, François is at D4-1010-5 and Maxime is at D4-1016-2.